pkg/dependency/credHelper/cred_helper_binary.go

```diff
@@ -0,0 +1,224 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package credHelper
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/opencontainers/go-digest"
+	"github.com/spf13/afero"
+
+	"github.com/runfinch/finch/pkg/command"
+	"github.com/runfinch/finch/pkg/config"
+	"github.com/runfinch/finch/pkg/dependency"
+	"github.com/runfinch/finch/pkg/flog"
+	"github.com/runfinch/finch/pkg/path"
+)
+
+type ConfigFile struct {
```

Comment:
Can we not just import this from the docker CLI project?

Reply:
Im not exactly sure how to import the struct only and not the entire package .